src/humidity.jl

```diff
@@ -0,0 +1,68 @@
+
+"""
+Humidity struct containing relevant parameters and arrays
+"""
+struct compute_relative_and_saturation_humidity{NF<:AbstractFloat}     
```

Comment:
If it's just a struct with constants why not defining them in `Constants`?

Reply:
Alternatively, we could collect them in a struct `HumidityConstants` (note that types, incl custom, i.e. primitive types or structs use camel font in Julia style) and use that in `Constants`. Hence we'd define and access them via

```julia
struct HumidityConstants
    e0::NF
    c1::NF
end

struct Constants
     humidity_constants = HumidityConstants(...)
end

function abc(C::Constants)
    @unpack e0 = C.humidity_constants
    ...
end
```

@unpack e0 = C.HumidityConstants